doc/source/conf.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+#
+# PIConGPU documentation build configuration file, created by
+# sphinx-quickstart on Tue Feb 14 17:29:56 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import subprocess
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.mathjax', 'sphinx.ext.githubpages', 'breathe']
+
+# breathe config
+breathe_projects = { 'PIConGPU': '../xml' }
+breathe_default_project = 'PIConGPU'
+
+breathe_domain_by_extension = {
+    "cu"       : "cpp",
+    "cpp"      : "cpp",
+    "kernel"   : "cpp",
+    "h"        : "cpp",
+    "hpp"      : "cpp",
+    "tpp"      : "cpp",
+    "def"      : "cpp",
+    "param"    : "cpp",
+    "unitless" : "cpp",
+    "loader"   : "cpp"
+}
+
+highlight_language = 'c++'
+
+# RTD
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+if on_rtd:
+    subprocess.call('cd ..; doxygen', shell=True)
+else:
+    import sphinx_rtd_theme
+    html_theme = "sphinx_rtd_theme"
+    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+source_suffix = ['.rst', '.md']
+#source_suffix = '.rst'
```

Comment:
OPTIONAL:
Here the comment does not show the options, but is a less detailed example of the above line. Could be removed, since the purpose of demonstrating usage is not fulfilled.